### PR TITLE
added some tests and increase coverage

### PR DESCRIPTION
**Pull request by sinisaos**
Some tests have been added, configuration and migration files have been omitted to increase coverage.

### CODE REVIEW

File: tests/session_auth/test_session.py [hunk 1]
```diff
@@ -370,3 +372,16 @@ def test_wrong_cookie_value(self):
             response.json(),
             {"is_unauthenticated_user": True, "is_authenticated": False},
         )
+
+
+class TestCleanSessions(TestCase):
+    def setUp(self):
+        SessionsBase.create_table().run_sync()
```
Comment (dantownsend) [inline tests/session_auth/test_session.py:379]: It would be nice here if we could add a session to the database table.

**Comment by dantownsend** [comment]
@sinisaos Thanks a lot for this - just one minor suggestion.

File: tests/session_auth/test_session.py (hunk 1 above)
Comment (sinisaos) [reply tests/session_auth/test_session.py:379]: @dantownsend I agree with you. I'll do it tomorrow.

Comment (dantownsend) [reply tests/session_auth/test_session.py:379]: Thanks!

**Comment by dantownsend** [comment]
@sinisaos Thanks again for this

**Pull request merged**